docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.3'
```

Comment:
Use the last version of docker-compose, please see: https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

```suggestion
version: '3.8'
```